src/mod/applications/mod_http_cache/mod_http_cache.c

```diff
@@ -768,7 +771,14 @@ static char *url_cache_get(url_cache_t *cache, http_profile_t *profile, switch_c
 
 		/* download the file */
 		url_cache_unlock(cache, session);
+		clock_gettime(CLOCK_REALTIME, &before);
```

Comment:
This code is not portable, please use switch_time_now instead

Reply:
Thanks! Fixed it. :)